prepare_data/gen_12net_hard_example.py

```diff
@@ -8,8 +8,8 @@
 import numpy as np
 import os
 from utils import *
-deploy = '12net.prototxt'
-caffemodel = '12net.caffemodel'
+deploy = '../demo/12net.prototxt'
+caffemodel = '../demo/12net.caffemodel'
```

Comment:
I think the original version is correct. The author hopes that we train the 12-net firstly, and then use 12-net trained by ourselves to generate hard examples of 24-net. The same is true of several other similar codes.
By the way, I have found other mistakes in this project.